Makefile

```diff
@@ -95,7 +88,6 @@ install: init ## install the package to the active Python's site-packages
 	pipenv run python setup.py install
 
 dev: init ## set up a development environment
```

Comment:
The suggestion was to use 'clean' for the dev target recently (https://github.com/release-depot/toolchest/pull/43)